src/simulation/ApplyLoad.cpp

```diff
@@ -943,6 +1101,243 @@ ApplyLoad::benchmark()
     closeLedger(txs, {}, /* recordSorobanUtilization */ true);
 }
 
+void
+ApplyLoad::findMaxLimitsForModelTransaction()
+{
+    uint64_t const INSTRUCTIONS_ROUNDING_STEP = 5'000'000;
```

Comment:
I see that this was done for readability of the limits and to make the binary search faster, but I'm curious if there was some methodology to choosing these numbers.